torch/_inductor/pattern_matcher.py

```diff
@@ -1616,7 +1616,26 @@ def is_start_of_fx_graph(graph: torch.fx.Graph, node: torch.fx.Node) -> bool:
 _mutation_op_re = re.compile(r"(?<!_)(_$|_[.]|(\b|_)(set|enter|exit|seed)(\b|_))(?!_)")
 
 
+def incorrect_inductor_schema_op(op: torch._ops.OpOverload) -> bool:
```

Comment:
nit: FIXME in function name?, with note in the TODO to not add anything here

Reply:
I have a pr after this stack that fixes them, but not sure if there are any interactions with functionalization that adding the mutation breaks cc @bdhirsh

Reply:
FWIW, i did open a pr updating, and things break... leaving as it is for now